src/components/badges/CircleCIBadge.tsx

```diff
@@ -0,0 +1,48 @@
+/* @jsx MD */
+import MD, { Component } from "jsx-md";
+import { Badge } from "../Badge";
+import { extractGithubOwnerAndRepo } from "./utils/extractGithubOwnerAndRepo";
+import { PackageJSON } from "../../PackageJSON";
+
+/** @internal */
+interface Props {
+  branch?: string;
+  style?: string;
+  pkg: Readonly<PackageJSON>;
+}
+
+/** Display a badge with the status of CircleCI Build */
+export const CircleCIBadge: Component<Props> = ({
+  branch,
+  style = "svg",
+  pkg,
+}) => {
+  if (style !== "svg" && style !== "shield") {
+    return null;
+  }
+  const ownerAndRepo = extractGithubOwnerAndRepo(pkg.repository);
+  if (ownerAndRepo === undefined) {
+    return null;
+  }
+  const { owner, repo } = ownerAndRepo;
+
+  if (branch === undefined) {
```

Comment:
This if/else can be simplified by using a ternay operator in lines 32/33 (see below)

Reply:
Actually, nevermind, I like this version better